.ci/magician/github/membership.go

```diff
@@ -22,15 +22,16 @@ var (
 		"roaks3",
 		"zli82016",
 		"trodge",
-		"hao-nan-li",
```

Comment:
You can stay here and only change the `onVacationReviewers` list.

Reply:
This is the reason for build-and-unit-tests failed.

Reply:
Thanks for catching it!